aspects/aspects.go

```diff
@@ -529,7 +514,7 @@ func get(subKeys []string, index int, node map[string]json.RawMessage, result in
 	rawLevel, ok := node[key]
 	if !ok {
 		pathPrefix := strings.Join(subKeys[:index+1], ".")
-		return &FieldNotFoundError{fmt.Sprintf("no value was found under %q", pathPrefix)}
+		return &NotFoundError{Cause: fmt.Sprintf("no value was found under %q", pathPrefix)}
```

Comment:
Nitpick: When this part of the error description is viewed within a complete error description e.g. [here](https://github.com/snapcore/snapd/pull/12953/files#diff-422229747513c03799364e913064839da6c339c01d5aeceb43f871ca94b83db6R66) it seems it misses a description of what %q is.

Reply:
Sorry, I missed this initially. I added a descriptor now